nav2_map_server/CMakeLists.txt

```diff
@@ -34,9 +34,8 @@ add_executable(${executable_name}
 set(library_name ${executable_name}_core)
 
```

Comment:
can we remove the yaml_cpp_vendor package from the CMakeLists now that the yaml files will be passed as parameters from the node?

Reply:
Yes, done.

Reply:
Awesome! That will help with CI as well